lib/components/term.js

```diff
@@ -299,6 +272,13 @@ export default class Term extends React.PureComponent {
 
   onTermWrapperRef(component) {
     this.termWrapperRef = component;
+
+    if (component) {
+      this.resizeObserver = new ResizeObserver(() => this.fitResize());
```

Comment:
```suggestion
      this.resizeObserver = new ResizeObserver(() => {
        if (this.resizeTimeout) {
          return;
        }
        this.resizeTimeout = setTimeout(() => {
          delete this.resizeTimeout;
          this.fitResize()
        }, 0);
      });
```

@chabou found an issue with the panes blinking in and out.

I'm not sure why it happens but it seems due to `fitResize` being called at the wrong part of the event loop. I tried wrapping it with `setTimeout(() => ..., 0)` and that seems to fix it (`requestAnimationFrame` did not)

![hyper-resize](https://user-images.githubusercontent.com/1410520/51417385-cb5ede00-1b4b-11e9-8eb6-41f73a4d6558.gif)